src/queues.c

```diff
@@ -134,15 +134,15 @@ int queues_notification_insert(struct notification *n)
         }
         /* Do not insert the message if it's a command */
         if (STR_EQ("DUNST_COMMAND_PAUSE", n->summary)) {
-                pause_displayed = true;
+                dunst_status(S_RUNNING, false);
```

Comment:
I don't think we'd ever need to actively signal a command line tool as it usually doesn't hang around, makes a request and exits.

But either case aside IMO it's best to avoid needless abstraction on a "might need this" basis, especially one as simple to implement later as this.

Reply:
I agree with you IMO, but I'm already working on #445 and to have the ability for a signal is mandatory for #216.

Also the status field is declared static in dunst.c.